config/locales/pl.yml

```diff
@@ -10,6 +10,7 @@ pl:
   destroy: 'Usuń'
   edit: 'Edycja'
   see: 'Zobacz'
+  remove_invoice: 'Usuń fakturę'
```

Comment:
I'm not sure if this is the correct place for this translation.
Where will you use it except in the maintenance form?

Reply:
maybe it should be change just for usuń?